MovingBlocks/TeraNUI
# build: upgrade gradle 6.5 -> 8.2.1

Relates to https://github.com/MovingBlocks/Terasology/pull/5109
Relates to https://github.com/MovingBlocks/Terasology/pull/4653

### Contains

* Updating gradle from 6.5 -> 8.2.1
* Updating gradle's files for which became invalid

### How to test

* run `gradlew build` -> should build TeraNUI
* we could further test it in combination with Terasology (`groovyw lib get TeraNUI && gradlew jar game`)

### Remarks

Java 20 is not yet supported, see https://github.com/gradle/gradle/issues/23488.

Closes #57
